ci-operator/step-registry/ipi/conf/azurestack/rhcos/ipi-conf-azurestack-rhcos-ref.yaml

```diff
@@ -12,11 +12,11 @@ ref:
     mount_path: /var/run/azurestack-cluster-secrets
   env:
   - name: RHCOS_VHD_RESOURCE_GROUP
-    default: "rhcos-storage-rg"
+    default: "rhcosvhd"
     documentation: |-
       The resource group of boot image vhd, the default value is for PPE3 environment.
```

Comment:
How about to update PPE3 -> WWT in documentation? same comments for documentation of `RHCOS_VHD_STORAGE_ACCOUNT`